### PR TITLE
Fix CircleCI run job

### PR DESCRIPTION
**Pull request by mbacchi**
When the initialize_dev_aliases step was added it required user interaction to succeed. This will always fail in CircleCI. We shouldn't default to this but rather allow developers to add these aliases manually if desired.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -306,7 +306,6 @@ if [ "$#" -eq "0" ]; then
   install_helm_plugins
   install_terraform
   initialize_nvm_auto_loading
-  initialize_dev_aliases
```
Comment (leonpham0) [inline bootstrap.sh:309]: Should we update [the README](https://github.com/codecademy-engineering/bootstrap/tree/f60d438f85c23e51e9fc629a9ad762d0a6541504#aliases) to make it more explicit that the aliases won't be setup by default and if you want them you should run some other command?

**Review by leonpham0 — APPROVED** [review]
I think this is fine, but left a note about updating the README to avoid any confusion.

File: bootstrap.sh (hunk 1 above)
Comment (mbacchi) [reply bootstrap.sh:309]: This is probably a good idea. Thanks I'll make that update soon.

**Comment by jakemhiller** [comment]
100% fine making these optional heh

File: bootstrap.sh (hunk 1 above)
Comment (mbacchi) [reply bootstrap.sh:309]: Note added to README. Let me know what you think @leonpham0.

**Pull request merged**